example/index.tsx

```diff
@@ -3,16 +3,24 @@ import { createContainer } from "../src/unstated-next"
 import { render } from "react-dom"
 
 function useCounter(initialState = 0) {
-	let [count, setCount] = useState(initialState)
-	let decrement = () => setCount(count - 1)
-	let increment = () => setCount(count + 1)
+	const [count, setCount] = useState(initialState)
```

Comment:
it's really unrelated  to change `let` => `const`, it only makes the pr hard to read, please do not do it!